apps/remix-ide-e2e/src/tests/solidityUnittests.test.ts

```diff
@@ -64,9 +64,8 @@ module.exports = {
       .waitForElementContainsText('*[data-id="testTabSolidityUnitTestsOutput"]', '✓ Initial value should be100', 120000)
       .waitForElementContainsText('*[data-id="testTabSolidityUnitTestsOutput"]', '✓ Value is set200', 120000)
       .waitForElementContainsText('*[data-id="testTabSolidityUnitTestsOutput"]', '✘ Should fail for wrong value200', 120000)
-      .waitForElementContainsText('*[data-id="testTabSolidityUnitTestsOutput"]', 'Passing: 2', 120000)
-      .waitForElementContainsText('*[data-id="testTabSolidityUnitTestsOutput"]', 'Failing: 1', 120000)
-      .waitForElementContainsText('*[data-id="testTabSolidityUnitTestsOutput"]', 'FAIL MyTest (tests/simple_storage_test.sol)', 120000)
```

Comment:
why is that test being removed? could you readd it?

Reply:
It was reading label fail as text `FAIL` previously, which is not the case now.

Reply:
that could be an issue.
Previous test reads `Failing: 1` so it means there's a failing test. the UI doesn't show `FAIL MyTest (tests/simple_storage_test.sol)` anymore?

Reply:
Is it possible to keep checking for the content `FAIL MyTest (tests/simple_storage_test.sol`, should be the same as you have the css class